src/env.ts

```diff
@@ -81,7 +81,7 @@ export const FALLBACK_LCD_ENDPOINT: Record<string, string> = {
 export const MAX_FILE_SIZE = 800_000;
 
 export const CELATONE_CONSTANTS: CelatoneConstants = {
-  gasAdjustment: 1.6,
+  gasAdjustment: 1.25,
```

Comment:
Let's use `Math.max` to set the upper bound after multiplying by `gasAdjustment` and leave gasAdjustment 1.6